lib/ReactViews/Preview/DataPreviewSections.jsx

```diff
@@ -58,19 +58,29 @@ const DataPreviewSections = observer(
         ? metadataItem.infoWithoutSources
         : metadataItem.info.slice();
 
+      const renderSection = item => {
+        let content = item.content;
+        try {
+          content = Mustache.render(content, metadataItem);
+        } catch (error) {
+          console.log(
+            `FAILED to parse info section ${item.name} for ${metadataItem.name}`
+          );
+          console.log(error);
+        }
+        return parseCustomMarkdownToReact(content, {
+          catalogItem: metadataItem
+        });
+      };
+
```

Comment:
This is here because a Mustache.render was throwing errors with info scraped from CSW.

We should probably handle this if we are parsing data from elsewhere